styleguide/source/_patterns/03-templates/04-topic/topic.twig

```diff
@@ -9,22 +9,25 @@
     </div>
```

Comment:
...besides the point, but <main> maybe needs to be <main id="main" class="topic" role="main" tabindex="-1"> I know we're not there yet, but had to say something. That "container" class is a wanton, right? Thoughts?

Reply:
Not sure what you mean by "wanton" :P but if you're talking about `.container` it's there to set a max-width - we made it its own class because sometimes we needed the grid without the container and vice versa

But I do agree we should add the other attributes wherever appropriate. That'd be a good standards checklist item?

Reply:
Also re: using ids - I've heard things for and against using IDs vs. classes without a really specific reason, what do you think?

Reply:
IMO IDs are good for use in JS, but should not be used in CSS if possible.